src/index.js

```diff
@@ -31,6 +32,8 @@ app.use('/public', express.static(path.join(__dirname, '../public')));
 
 routes(app);
 
+// app.use('/api/v1', routes);
```

Comment:
can you please remove this unnecessary comment

Reply:
done